cmd/fs-v1-helpers.go

```diff
@@ -424,3 +457,18 @@ func fsRemoveMeta(basePath, deletePath, tmpDir string) error {
 	}
 	return fsDeleteFile(basePath, deletePath)
 }
+
+// List all buckets at meta bucket prefix in `.minio.sys/buckets/` path.
+// This is implemented to avoid a bug on windows with using readDir().
+func fsReaddirBuckets(fsPath string) ([]string, error) {
```

Comment:
Note that this function gets used only for migration. Not during regular FS operations. fsReaddirBuckets reads `fsPath/.minio.sys/buckets` not `fsPath` hence fsReaddirMetaBuckets sounds more appropriate.

Also better move this closer to `migrateFSObject()`